kubernetes/chart/vault-ui/values.yaml

```diff
@@ -0,0 +1,42 @@
+# Default values for vault-ui.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: djenriquez/vault-ui
+  tag: latest
```

Comment:
`latest` is a better default that a certain tag. When installing or upgrading the application, Helm provides different ways of setting or overriding these values.

The chart allows you to install the application, which version of the application you want to run it's a parameter that you can inject/override.

For example, you would run `latest` to test that the app does what you want, but if you want to run it in prod, you will run a certain tag.